openedx/features/edly/signals.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from lms.djangoapps.instructor.enrollment import get_email_params
+from openedx.features.edly.tasks import send_course_enrollment_mail
+from student.models import CourseEnrollment
+
+
+@receiver(post_save, sender=CourseEnrollment)
+def handle_user_enroll(sender, instance, **kwargs):
+    """
+
+    :param sender: Model from which we received signal.
+    :param instance: Instance of model which has been created or updated
+    :param kwargs: Remaining parts of signal.
+    """
+
+    # This if condition check's for if the Enrollment signal is created/updated using LMS or not.
```

Comment:
it might be
`Check if the Enrollment signal is created/updated using LMS or not.

Reply:
?